packages/nimble-components/build/generate-workers/source/renderWorker.ts

```diff
@@ -0,0 +1,67 @@
+import * as Comlink from 'comlink';
+
+export class RenderWorker {
+    private canvas!: OffscreenCanvas;
+    private worker!: number;
+    private context!: OffscreenCanvasRenderingContext2D;
+    private dieMatrix: {
```

Comment:
There is still some debate about the matrix API, can you please set it as a simple UInt8Array for testing purposes only? we will change it accordingly after the debate is settled